src/main/java/trackr/model/order/Order.java

```diff
@@ -0,0 +1,106 @@
+package trackr.model.order;
+
+import static trackr.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import trackr.model.order.customer.Customer;
+
+/**
+ * Represents an Order in the order list.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Order {
+
+    //Data fields
+    private final OrderName orderName;
```

Comment:
Is the food name and quantity going to be the name of the order instead of being seperate?